src/resolvers/helpers/filter.js

```diff
@@ -113,18 +108,11 @@ export function filterHelper(
       resolveParams.query = resolveParams.query.where({ _id: { $in: _ids } });
     }
     processFilterOperators(filterFields);
-    const clearedFilter = {};
-    Object.keys(filterFields).forEach((key) => {
-      if (modelFields[key] || key.indexOf('$') === 0) {
-        clearedFilter[key] = filterFields[key];
-      } else if (aliases && aliases?.[key]) {
-        clearedFilter[aliases[key]] = filterFields[key];
-      }
-    });
-    if (Object.keys(clearedFilter).length > 0) {
-      // eslint-disable-next-line
-      resolveParams.query = resolveParams.query.where(toMongoFilterDottedObject(clearedFilter));
-    }
+
+    // eslint-disable-next-line
+    resolveParams.query = resolveParams.query.where(
+      toMongoFilterDottedObject(resolveParams.query.model.translateAliases(filterFields))
```

Comment:
Okay, `translateAliases` is a pretty cool trick I found

Reply:
Didn't know about this helper method. I'd already implement my own solution https://github.com/graphql-compose/graphql-compose-mongoose/blob/master/src/resolvers/helpers/filter.ts#L109-L115 But it's ugly. Built-in mongoose solution I like more.